packages/block-library/src/image/image.js

```diff
@@ -426,7 +426,14 @@ export default function Image( {
 
 	const sizeControls = (
 		<InspectorControls>
-			<ToolsPanel label={ __( 'Settings' ) } resetAll={ resetAll }>
+			<ToolsPanel
+				label={ __( 'Settings' ) }
+				resetAll={ resetAll }
+				dropdownMenuProps={ {
+					placement: 'left-start',
+					offset: 258, // sidebar width (280px) - button width (24px) + border (2px)
+				} }
```

Comment:
It would be nice to refactor these to a `TOOLSPANEL_DROPDOWNMENU_PROPS` constant like done for files in the block editor package.

Also interesting to spot a difference in the offset (258 vs 259), I wonder if they should be the same?